wfe/web-front-end.go

```diff
@@ -227,7 +227,7 @@ func (wfe *WebFrontEndImpl) NewAuthorization(response http.ResponseWriter, reque
 
 	body, key, err := verifyPOST(request)
 	if err != nil {
-		wfe.sendError(response, "Unable to read/verify body", http.StatusBadRequest)
+		wfe.sendError(response, fmt.Sprintf("Unable to read/verify body: %v", err), http.StatusBadRequest)
```

Comment:
Agreed, I've reverted to the previous behavior and also changed `NewRegistration` to follow this behavior for now.